Python 3.10.4 (tags/v3.10.4:9d38120, Mar 23 2022, 23:13:41) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

= RESTART: C:/Users/user/AppData/Local/Programs/Python/Python310/bayesclassifier1.py
Enter number of dolomite cores: 78
Enter number of Shale cores: 34
Enter the mean of dolomite: 54
Enter the mean of Shale: 32
Enter the standard Dev of dolomite: 14
Enter the standard Dev of Shale: 79
Traceback (most recent call last):
  File "C:/Users/user/AppData/Local/Programs/Python/Python310/bayesclassifier1.py", line 13, in <module>
    P_S = No_of_Shale/Total
NameError: name 'No_of_Shale' is not defined

= RESTART: C:/Users/user/AppData/Local/Programs/Python/Python310/bayesclassifier1.py
Enter number of dolomite cores: 78
Enter number of Shale cores: 34
Enter the mean of dolomite: 54
Enter the mean of Shale: 32
Enter the standard Dev of dolomite: 14
Enter the standard Dev of Shale: 79
P(D) = 0.6964285714285714
P(S) = 0.30357142857142855
P(gamma>60(D) = 0.3341175708976246
P(gamma>60(S) = 0.3615081852878814
The interval is dolomite
